demo-automation-java/src/test/java/com/example/tests/BaseTest.java

```diff
@@ -18,27 +18,18 @@ public class BaseTest {
     public WebDriverWait wait;
 
     @BeforeMethod
-    public void setUp() throws Exception {
-<<<<<<< Updated upstream
-                
-        try {            
+    public void setUp()   {
+
+        try {
             config = new Configuration("src/test/java/com/example/core/configuration/config.properties");
             driver = DriverFactory.getDriver(config.getProperty("browser"));
-            keyword = new WebKeyword(driver);            
+            keyword = new WebKeyword(driver);
             keyword.openUrl(config.getProperty("url"));
             int implicitWait = Integer.parseInt(config.getProperty("implicitWait"));
             this.wait = new WebDriverWait(driver, implicitWait);

```

Comment:
Are you apply implicit wait in here?
But I haven't seen implicit applied here, it could be "driver.manage().timeouts().implicitlyWait(implicitWait, TimeUnit.SECONDS);" insted "new WebDriverWait".

Reply:
got it